src/pages/fundamentals/eutxo-patterns.mdx

```diff
@@ -29,23 +29,231 @@ the transaction contain the corresponding `Input`. By doing this, the minting po
 validate a single time. In some designs, this logic is used for only a subset of redeemers to allow more
 flexible minting policies.
 
+
+#### NFT example:
```

Comment:
The 'receipts' pattern below is also for NFTs, NFTs do not need a one-shot minting policy, it's just one way to achieve uniqueness.

Let's just change the contents of this comment to `Example:` potentially?